Sources/DolbyIORTSCore/Manager/RendererRegistry.swift

```diff
@@ -37,7 +40,8 @@ final actor RendererRegistry: RendererRegistryProtocol {
 
     func deregisterRenderer(_ renderer: StreamSourceViewRenderer, for track: MCVideoTrack) {
         guard let trackID = track.getId() else {
-            fatalError("Expect to have dictionary entry for the track before querying for renderer")
+            Self.logger.error("📺 Deregister renderer \(renderer.id) called with an invalid VideoTrack")
```

Comment:
Unregister?

Reply:
Isn't both deregister and unregister the same meaning ?

Reply:
NW. I couldn't find deregister in the 1st dictionary I used, but I found it in the 2nd one.